tests/acm_observability/conftest.py

```diff
@@ -0,0 +1,130 @@
+import json
+import os
+import shlex
+
+import pytest
+import requests
+from const import HUB_CLUSTER
+from ocp_resources.multi_cluster_observability import MultiClusterObservability
+from ocp_resources.route import Route
+from ocp_utilities.utils import run_command
+from pytest_testconfig import config as py_config
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="session")
+def multi_cluster_observability(admin_client_scope_session):
+    observability = MultiClusterObservability(
+        client=admin_client_scope_session,
+        name="observability",
+    )
+    assert (
+        observability.exists
+    ), f"{observability.name} MultiClusterObservability does not exist"
+    observability.wait_for_condition(
```

Comment:
5 seconds are enough?

Reply:
for running the tests it is required that MultiClusterObservability resource is available and ready (for whoever consumes these tests).
`wait_for_condition` is only used for asserting that status

Reply:
`observability.status` is not enough?

Reply:
that was my initial approach, @myakove suggested it will be better to use `wait_for_condition`

Reply:
@rnetser what do you mean?
Wait for condition ready to be true
This indicate that it is up and ready.